src/js/structs/Job.js

```diff
@@ -127,6 +128,6 @@ module.exports = class Job extends Item {
   }
 
   toJSON() {
-    return this.get();
+    return cleanJobJSON(this.get());
```

Comment:
Side note: We're going to remove the clean methods once we've  separated the status from definition/spec data.